<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportControlParameters</class>
 <widget class="QWidget" name="ImportControlParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView_import_control_parameters">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>801</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_load_files">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>330</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Načíst znovu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close_import_control_parameters">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>330</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Zavřít</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_import_parameters">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>360</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Control parameters</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_import_input_channels">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>360</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Parametry snímačů</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_import_level">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>360</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Level</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_import_tolerance">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>360</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Tolerance band</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_import_info">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>360</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Informace o testu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Importovat:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_import_profile_from_test_file">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Profil testu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_import_whole_file">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>360</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Načíst celý soubor</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Vyhledat podle:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_import_filter_year">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>roku</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>firmy</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_import_filter_company_name">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_close_import_control_parameters</sender>
   <signal>clicked()</signal>
   <receiver>ImportControlParameters</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>739</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>737</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
